core/unix/os.c

```diff
@@ -6374,11 +6396,10 @@ handle_exit(dcontext_t *dcontext)
             exit_process ? "process" : "thread");
     }
     KSTOP(num_exits_dir_syscall);
-
```

Comment:
nit: No reason to remove the blank line which was setting this off from the block above, and is symmetric with the blank before that block

Reply:
Fixed.